editions/tw5.com/tiddlers/widgets/Startup Actions

```diff
@@ -0,0 +1,19 @@
+tags: ActionWidgets
+title: Startup Actions
+
+Any tiddler that is tagged with `$:/tags/StartupAction` will have the contents of its text field evaluated as though it were triggered by a button widget. That is, any [[action widgets|ActionWidgets]] will be invoked.
+
+Any WikiText markup is allowed, so transclusions and macros work as expected.
+
+!Example
+
+The following script, if put in the text field of a tiddler tagged with `$:/tags/StartupAction`, would change the title of a wiki to `title1` if it is not already `title1`, and if the title of the wiki is `title1` it will change the title to `title2`. This is done using the [[reveal widget|RevealWidget]].
```

Comment:
imo use "wikitext" instead of script. ... If I read script, it's something that needs to be sanitized, because it may contain evil content.